app/src/ui/app.tsx

```diff
@@ -583,6 +583,27 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setAppMenuState(menu => menu.withReset())
   }
 
+  private onTitlebarDoubleClick() {
+    if (__DARWIN__) {
+      const actionOnDoubleClick = remote.systemPreferences.getUserDefault('AppleActionOnDoubleClick', 'string')
+      const mainWindow = remote.getCurrentWindow()
+
+      if (mainWindow) {
+        switch (actionOnDoubleClick) {
+          case 'Maximize':
+          default:
```

Comment:
Do we need this `default` here? I'm curious if there's a value here that's not `Maximize | Minimize | None` that we might encounter.